packages/vuepress-theme-nimatron/styles/_samples.scss

```diff
@@ -163,50 +184,19 @@ $spacing-units: (
   }
 }
 
-.sample--colors-wrap {
-  /* Fallback Support */
-  display: flex;
-  display: grid;
-  grid-gap: $spacing-m;
-  grid-template-columns: repeat(auto-fill, minmax(($max-line-length / 2), 1fr));
-  flex-wrap: wrap;
-}
-
-.sample--color-list {
-  margin-bottom: $spacing-l;
-  overflow: hidden;
+.sample-color {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  margin: 0 auto;
   border-radius: $base-border-radius;
-}
-
-.sample--color {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  margin-bottom: 0;
-  padding: $spacing-xs $spacing-s;
+  vertical-align: middle;
 }
 
 @each $hue, $variants in $colors {
   @each $variant, $color in $variants {
-    .is-sample-#{$hue}-#{$variant} {
+    .sample-color--#{$hue}-#{$variant} {
       background: cv($hue, $variant);
-
-      &::before,
-      &::after {
-        position: static;
-        padding: $spacing-xs;
-        background: $white;
-        font-size: $size-body-caption;
-        font-weight: 400;
-      }
-
-      &::before {
-        content: '#{$hue}, #{$variant}';
-      }
-
-      &::after {
-        content: '#{$color}';
```

Comment:
this was a really clever away to approach this problem, and still, I didn't quite like it. RIP.

Reply:
Same - never quite lived up to itself 😭